@here/harp-mapview/lib/MapView.ts

```diff
@@ -2713,30 +2713,35 @@ export class MapView extends THREE.EventDispatcher {
         if (this.m_drawing) {
             return;
         }
+
         ++this.m_frameNumber;
+        const frameStartTime = time;
 
+        let currentFrameEvent: FrameStats | undefined;
         const stats = PerformanceStatistics.instance;
         const gatherStatistics: boolean = stats.enabled;
+        if (gatherStatistics) {
+            currentFrameEvent = stats.currentFrame;
```

Comment:
why was this moved here, can't it be part of the if(gatherStat...) on line 2742?

Reply:
I think the perf test gathers the statistics in the `RENDER_EVENT. Will also double check this.

Reply:
the perftests are listening to the `AfterRender` event.So I move the dispatching of the render event to the very top (and also don't count it into the frame time b/c it's not rendering but application code)